code/game/objects/items/weapons/grenades/flashbang.dm

```diff
@@ -56,18 +56,34 @@
 				M.Confused(status_duration * 2)
 
 		// Bang
-		var/ear_safety = M.check_ear_prot()
 		if(bang)
+			var/ear_safety = M.check_ear_prot()
```

Comment:
Can you reduce the indents here where possible via guard clauses?
```suggestion
		if(!bang)
			return
		var/ear_safety = M.check_ear_prot()
```

Reply:
whoops! Sure.